src/puppetlabs/puppetdb/mq_listener.clj

```diff
@@ -259,8 +260,9 @@
   [{:keys [command annotations] :as msg} e discard]
   (let [attempt (count (:attempts annotations))
         id      (:id annotations)
+        certname (get-in msg [:payload :certname])
         msg     (annotate-with-attempt msg e)]
-    (log/errorf e "[%s] [%s] Fatal error on attempt %d" id command attempt)
+    (log/error e (i18n/trs "[{0}] [{1}] Fatal error on attempt {2} for ''{3}''" id command attempt certname))
```

Comment:
minor point, but it may be more consistent not to quote the certname in this message. This is how we log when command processing succeeds:

```
2016-05-03 11:57:47,931 INFO  [p.p.command] [5808e1cb-5469-459a-af6d-f5e1e27833f3] [replace facts] mbp.local
```